src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/DependenciesTreeSearchProvider.cs

```diff
@@ -43,6 +44,8 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedColl
     /// </para>
     /// </remarks>
     [Export(typeof(ISearchProvider))]
+    [Name("DependenciesTreeSearchProvider")]
+    [VisualStudio.Utilities.Order(Before = "GraphSearchProvider")]
```

Comment:
Should this also include `After = "HierarchySearchProvider"`?

Right now it seems underspecified, given the intention is to have the order be: hierarchy / extensions / graph.

Reply:
From Hugues comment on the other PR, seems like the current ordering should be fine as long as GraphProvider is the last one.

Reply:
Ok that's fine. It was the description on the other PR that concerned me. So long as this is understood. I'll merge this now. Thanks @ketttY.